apps/services/auth/public-api/src/app/modules/delegations/actorDelegations.controller.spec.ts

```diff
@@ -108,8 +118,33 @@ describe('ActorDelegationsController', () => {
     })
 
     describe('GET /actor/delegations', () => {
+      let nationalRegistryApiSpy: jest.SpyInstance
       const path = '/v1/actor/delegations'
       const query = '?direction=incoming'
+      const deceasedNationalIds = generateItemFromCnt(3, getFakeNationalId)
```

Comment:
Can use existing utility function from `@island.is/testing/fixtures` to generate better fake national ids.